src/api/app/controllers/concerns/triggerable.rb

```diff
@@ -27,6 +28,7 @@ def set_package
     # TODO: This should not happen right? But who knows...
     raise ActiveRecord::RecordNotFound unless @package
   end
+  # rubocop:enable Naming/AccessorMethodName
```

Comment:
This comment could be right under `def set_package(package_find_options:)`.

Reply:
Done